### DISCUSSION TITLE
FREE THE HOSTAGES

### DISCUSSION THREAD

**Discussion by jdsartwrh** [shaked6540/YoutubePlaylistDownloader#306] [General]
FREE THE HOSTAGES

**Comment by shaked6540** [comment]
Thanks for your support

**Reply by alfabe78h** [reply]
Arent there 5 thousand Palestinian hostages in Israeli detention camps?

**Reply by shaked6540** [reply]
There 5 thousand Palestinian criminals in Israeli jail, stabbing, shooting, burning and raping is illegal, so they are in jail. The Israeli hostages were kidnapped from their home in the October 7 attacks on Israel, they are normal civilians who did no crime, especially not in Gaza since no Israeli has been in Gaza for the past 20 years. Besides, 2 of them are babies, less than 1 year old when they were kidnapped, go ahead, try to justify that

**Comment by dardizzola** [comment]
I believe no Israeli or Palestinian authorities frequent GitHub.
You've been a pain in the ass with your messages...
This is a work environment where we talk about technology, not politics!
